.github/actions/latest-stable-tag/get-latest-tag.js

```diff
@@ -0,0 +1,41 @@
+export function getLatestTag(tags) {
+  const sortedTags = tags
+    .filter(tag => !isPreRelease(tag))
+    .sort(compareTagsDescending);
+  const latestTag = sortedTags[0];
```

Comment:
Out of curiosity, are you declaring the variables as a form of documentation?

Reply:
Probably... I wrote that a while ago, I don't remember what I was thinking!